docs/local-setup/running-testnet.md

```diff
@@ -28,42 +33,18 @@ Follow next instructions to install Docker on your machine:
 
 The following instructions will only work once you're running Docker on your machine.
 
-_NOTE: We don't recommend this unless you're contributing to `nearcore` or you know what you're trying to do, but you can run a node without docker by adding the `--nodocker` flag to the start testnet script found below. See how to do this under_ [_Compile TestNet without Docker_](/docs/local-setup/running-testnet)_._
+_NOTE: We don't recommend this unless you're contributing to `nearcore` or you know what you're trying to do, but you can run a node without docker by adding the `--nodocker` flag to the start testnet script found below. See how to do this under_ [_Compiling and Running Official Node (testnet/betanet) without Docker ](/docs/local-setup/running-testnet)_._
 
-## Running official TestNet node with Docker
+## Running Official Node (testnet/betanet) with Docker
 
-To run locally, clone the `nearcore` repo.
+Once nearup and Docker are installed (by following instrcutions in previous section), just run:
 
-```bash
-git clone https://github.com/nearprotocol/nearcore.git
-```
-
-`cd` into it...
-
-```bash
-cd nearcore
 ```
-
-`checkout` to latest development version of start script:
-```bash
-git checkout stable
+nearup {testnet, betanet}
```

Comment:
brace expansion may lead to confusion

Reply:
Changed the sample commands to `betanet` across the document. Added the information towards the top of the document that the instructions throughout the document use `betanet` in sample commands. Additional instructions added to run TestNet using `testnet` wherever applicable.